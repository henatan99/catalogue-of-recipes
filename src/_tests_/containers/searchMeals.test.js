import React from 'react';
import { Provider } from 'react-redux';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import SearchMeals from '../../containers/searchMeals';
import createTestStore from './testStore';

describe('SearchMeals', () => {
  let store;
  beforeEach(() => {
    store = createTestStore();
  });
  test('should match with snapshot', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <MemoryRouter>
            <SearchMeals />
          </MemoryRouter>
        </Provider>,
      ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
