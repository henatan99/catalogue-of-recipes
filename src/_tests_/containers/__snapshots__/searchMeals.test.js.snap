// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Meal should match with snapshot 1`] = `<div />`;

exports[`SearchMeal should match with snapshot 1`] = `<div />`;

exports[`SearchMeals should match with snapshot 1`] = `<div />`;
